.github/workflows/test.yml

```diff
@@ -45,30 +45,30 @@ jobs:
             (echo; echo "Unexpected difference in directories after code generation. Run 'go generate ./...' command and commit."; exit 1)
 
   # Run acceptance tests in a matrix with Terraform CLI versions
-  test:
```

Comment:
Commenting the acceptance tests out for now -- they weren't doing anything before anyhow. We'll need to stub our API somehow to get these working